packages/grafana-data/src/datetime/timezones.ts

```diff
@@ -148,13 +152,13 @@ const abbrevationWithoutOffset = (abbrevation: string): string => {
 
 const mapToInfo = (timeZone: TimeZone, timestamp: number): TimeZoneInfo | undefined => {
   const momentTz = moment.tz.zone(timeZone);
-
   if (!momentTz) {
     return undefined;
   }
 
   return {
     name: timeZone,
+    ianaName: momentTz.name,
```

Comment:
I think the proper name is olsonId.

Reply:
Did a quick check on this and seems like there is no clear winner;) I think I'll stay with iana as [Wikipedia states Olson database is occasionally used in as a name of this DB](https://en.wikipedia.org/wiki/Tz_database)